src/lib/containers/FilePreview/HtmlPreview/HtmlPreview.tsx

```diff
@@ -40,12 +40,30 @@ export type HtmlPreviewProps = {
  */
 export default function HtmlPreview(props: HtmlPreviewProps) {
   const { createdByUserId, rawHtml } = props
+  const frameEl = useRef(null)
+  const [frameHeight, setFrameHeight] = useState(100)
 
   const { data: teamMembership, isLoading } = useGetIsUserMemberOfTeam(
     TRUSTED_HTML_USERS_TEAM_ID,
     createdByUserId,
   )
-
+  const updateHeight = () => {
+    if (frameEl?.current && frameEl?.current['contentWindow']) {
+      let newHeightPx: number =
+        frameEl.current['contentWindow']['document']['body']['scrollHeight']
+      if (newHeightPx < 450) {
+        newHeightPx = 450
+      }
+      if (!frameHeight || Math.abs(newHeightPx - frameHeight) > 70) {
+        setFrameHeight(newHeightPx + 50)
+      }
+    }
+  }
+  useEffect(() => {
+    setInterval(() => {
+      updateHeight()
+    }, 500)
+  }, [])
```

Comment:
Should probably rerun this effect if the raw HTML changes in case we do something funny, like pass an empty string and change it out after the resource loads.

```suggestion
  }, [rawHtml])
```